build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -585,6 +585,47 @@ export const handlers: Handler[] = [
             return { resolved: resolved };
         },
     },
+    {
+        name: "npm-package-json-prettier",
+        match,
+        handler: (file) => {
+            let json;
+
+            try {
+                json = JSON.parse(readFile(file));
+            } catch (err) {
+                return "Error parsing JSON file: " + file;
+            }
+
+            const hasPrettierProperty = Object.prototype.hasOwnProperty.call(json, "prettier");
+            const hasPrettierFixProperty = Object.prototype.hasOwnProperty.call(
+                json,
+                "prettier:fix",
+            );
+            const hasFormatProperty = Object.prototype.hasOwnProperty.call(json, "format");
+
+            if (!(hasPrettierProperty && hasPrettierFixProperty && hasFormatProperty)) {
+                //return "Error parsing JSON file: " + file;
+                const missingScripts: string[] = [];
+
+                if (hasPrettierProperty === false) {
+                    missingScripts.push(`prettier`)
+                }
+
+                if (hasPrettierFixProperty === false) {
+                    missingScripts.push(`prettier:fix`)
+                }
+
+                if (hasFormatProperty === false) {
+                    missingScripts.push(`format`)
+                }
+
+                return file + ` missing following scripts in package.json` + missingScripts.join("\n");
```

Comment:
Nit:
```suggestion
                return `${file} is missing the following scripts: ${missingScripts.join("\n\t")}`;
```

We generally recommend using the string interpolation syntax where possible. This file isn't doing a great job at that, but we might as well us it with new code.

I'm also suggesting adding a tab before each newline entry to make the formatting a bit nicer, but feel free to ignore / do what you think looks best :)

Reply:
Ah, I missed using string interpolation. Thank you for letting me know 👍